openwisp_controller/config/tests/test_admin.py

```diff
@@ -916,6 +916,11 @@ def test_clone_template(self):
         self.assertContains(response, '{} (Clone 2)'.format(t.name))
         response = self.client.post(path, data, follow=True)
         self.assertContains(response, '{} (Clone 3)'.format(t.name))
+        path = reverse('admin:index')
+        response = self.client.get(path)
+        self.assertIn('test-template (Clone 3)', str(response.content))
+        self.assertIn('test-template (Clone 2)', str(response.content))
+        self.assertIn('test-template (Clone)', str(response.content))
```

Comment:
this is nice, but please assert also the presence of LogEntry objects in the database as suggested by @pandafy because it's 100% reliable